docs/features/drafts/lich/folios/coding.md

```
@@ -0,0 +1,35 @@
+status: draft
```

markdownlint-cli2 (0.23.0)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)